### PR TITLE
exercise 1

### PR DESCRIPTION
**Pull request by DonYeh**
-creates DOM elements for city name, temp, and wind speed
-shows weather icon
-shows map

### CODE REVIEW

File: scripts/index.js [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
 
 // Implement the following functions to fulfill the tests!
-function getLocationName() {}
-function getLocationCountry() {}
-function getLocationLatitude() {}
-function getLocationLongitude() {}
-function getDescription() {}
-function getWindSpeed() {}
-function getSunrise() {}
+function getLocationName(weather) {
```
Comment (antdevelopment1) [inline scripts/index.js:4]: Nice use of Map!!!

File: scripts/index.js [hunk 2]
```diff
@@ -1,25 +1,41 @@
...
+  const city = new Map();
+  const cityValues = Object.values(weather); // array of city values
+  return cityValues[10]; //city name is in index 10
+}
 
+function getLocationCountry(weather) {
```
Comment (antdevelopment1) [inline scripts/index.js:13]: Clean solutions and formatting. Great job guys!!!

**Review by antdevelopment1 — APPROVED** [review]
Great teamwork. Looks really nice!!!

File: index.html [hunk 3]
```diff
@@ -1,14 +1,25 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Document</title>
-</head>
-<body>
-    
+  </head>
+  <body>
+    <h1>Weather app</h1>
+    <div class="cityNameDiv" data-city></div>
```
Comment (antdevelopment1) [inline index.html:11]: Nice descriptive names for your divs.

File: scripts/index.js [hunk 4]
```diff
@@ -4,6 +4,9 @@
 function getLocationName(weather) {
   const city = new Map();
   const cityValues = Object.values(weather); // array of city values
+  const cityName = document.querySelector("[data-city]");
+  cityName.textContent = `city name: ${cityValues[10]}`;
+  body.appendChild(cityName);
```
Comment (antdevelopment1) [inline scripts/index.js:9]: Good job getting the element to the div using append.

File: scripts/index.js [hunk 5]
```diff
@@ -21,12 +24,76 @@ function getDescription(weather) {
...
+}
+function showIcon(iconURL) {
+  const weatherDiv = document.createElement("img");
+  // weatherDiv.textContent = `weather icon will be displayed here: ${iconURL}`;
+  weatherDiv.setAttribute("src", iconURL);
+  body.appendChild(weatherDiv);
+}
+
+function showMap(weather) {
+  const iFrameMap = document.querySelector("[data-map");
```
Comment (antdevelopment1) [inline scripts/index.js:67]: Awesome job pulling in google maps!!!

File: scripts/index.js [hunk 6]
```diff
@@ -21,12 +24,76 @@ function getDescription(weather) {
...
+  windSpeed.textContent = `wind speed: ${weather.wind.speed}`;
+  body.appendChild(windSpeed);
   return weather.wind["speed"];
 }
 function getSunrise(weather) {
   return weather.sys["sunrise"];
 }
 
+function getLocationTemp(weather) {
+  const cityTemp = document.querySelector("[data-temp]");
```
Comment (antdevelopment1) [inline scripts/index.js:37]: Great consistent use with storing data attributes!!!

File: scripts/weather.js [hunk 7]
```diff
@@ -1,7 +1,31 @@
-const atlWeather = {"coord":{"lon":-84.39,"lat":33.75},"weather":[{"id":800,"main":"Clear","description":"clear sky","icon":"01d"}],"base":"stations","main":{"temp":272.76,"pressure":1022,"humidity":40,"temp_min":270.15,"temp_max":275.37},"visibility":16093,"wind":{"speed":3.1,"deg":340},"clouds":{"all":1},"dt":1551797654,"sys":{"type":1,"id":4399,"message":0.0072,"country":"US","sunrise":1551787231,"sunset":1551829066},"id":4180439,"name":"Atlanta","cod":200};
+const atlWeather = {
+  coord: { lon: -84.39, lat: 33.75 },
```
Comment (antdevelopment1) [inline scripts/weather.js:2]: Great idea breaking up the data structure. It makes it lot easier to read. :)

**Review by antdevelopment1 — APPROVED** [review]
Great job guys!!!

**Review by antdevelopment1 — APPROVED** [review]

**Review by antdevelopment1 — APPROVED** [review]

**Comment by radishmouse** [comment]
thanks for the great code review @antdevelopment1 !

**Pull request closed without merging**